### DISCUSSION TITLE
Comparison with other tools

### DISCUSSION THREAD

**Discussion by tomatau** [expressots/expressots#503] [General]
### Is there an existing proposal similar to this?

- [X] I have searched the existing proposals

### What are you proposing?

It would be helpful when choosing a framework like this to compare with the existing ecosystem. It's always a gamble to choose a "new" framework so it helps to lay the ground clearly about how this compares with other existing competition - in this case, frameworks like Nest and Adonis come to mind

### Is there any specific group of users that will benefit from this?

Anyone performing discovery on which tooling to use

### What problems are you trying to solve?

I'm trying to make the pros and cons of this tool more discoverable

### Do you have any references or examples that can illustrate your idea?

There are many comparisons of existing frameworks covering things like:
- Features
- Benchmarks
- Integrations
- Community
- Etc..

### What type of idea is this?

Improvement of existing idea: Similar idea exists but this is an improvement

**Comment by rsaz** [comment]
Hi @tomatau thank you for opening the issue, your point is valid.

Right now we are looking for people to create content about ExpressoTS, either Youtube, Twitter, Dev.to, Medium. These advocates or content creators will significantly clarify the project goal as well as to expand its presence around the globe.

As framework creator, with limited time and resources available I need to focus energy in few things at the time. My current priorities are to deliver few other tool that are crucial for more complex backend scenarios.

In the next couple of months we will dedicate a huge portion of our time to make the framework well known and understood in terms of capabilities and how its compare with against their competitors.

Here a [technical comparison ](https://github.com/expressots/expressots/wiki/Benchmark) we did months ago and we already outperform Nestjs. Adonisjs is a basically stale product that probably will not continue to push the boundaries forward. The big real competitor in the Typescript space is Nestjs.

In terms of differences, we have a complete different philosophy and all the mechanics within the  tool are confirming and affirming the framework philosophy. We aim simplicity (and to make it simple, requires complex features). Simplicity for the developer, to code their company next project in a framework that is fully featured, that has a natural and easy way to structure the project and maintaining and so on.

Would be lovely to see people like yourself pushing ExpressoTS forward. You may end up in a page like this: https://expresso-ts.com/team

We will create another section/page for content creators to tag their content. :)
